projects/views.py

```diff
@@ -155,3 +158,26 @@ def get(self, request, *args, **kwargs):
 
         serializer = self.get_serializer(queryset, many=True)
         return Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class ProjectGetControlList(generics.ListAPIView):
+    queryset = Project.objects.all()
+    lookup_url_kwarg = "project_id"
+    serializer_class = ProjectControlListSerializer
+    filter_backends = (filters.DjangoFilterBackend,)
+    pagination_class = PageNumberPagination
+
+    def list(self, request, *args, **kwargs):
+        project_id = self.kwargs["project_id"]
+        project = self.get_object()
+        queryset = self.filter_queryset(
+            project.controls.through.objects.filter(project_id=project_id).order_by(
+                "control_id"
+            )
+        )
```

Comment:
I'm not sure that we need to grab the through table fields in this way, but we can circle back around and improve.